### PR TITLE
feat: added tests and standardized configuration

### PR DESCRIPTION
**Pull request by arkadiusz-czerwinski**
## What type of PR is this?

feature

## What this PR does / why we need it:

This PR changes versioner's config to standard convention and adds detailed tests for uploading datasets.

## Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

**Comment by Shanduur** [comment]
Fix commit messages.

File: data/tests/versioning/test_versioner.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+from os import path
+from pathlib import Path
+from data.versioning.versioner import WandbDatasetVersioner, VersioningConfig
```
Comment (Shanduur) [inline data/tests/versioning/test_versioner.py:3]: Do we need anything besides `WANDB_API_KEY` in CI env to make this work?

**Review by Shanduur — CHANGES_REQUESTED** [review]
Fix commit messages.

File: data/tests/versioning/test_versioner.py (hunk 1 above)
Comment (arkadiusz-czerwinski) [reply data/tests/versioning/test_versioner.py:3]: No

**Comment by Fakser** [comment]
Seems like wandb key is not provided, so ci will fail always
image

**Review by Fakser — CHANGES_REQUESTED** [review]
Please add proper wandb key
image

**Review by damiankucharski — APPROVED** [review]

**Comment by Fakser** [comment]
This PR is included in [https://github.com/PolSl-PBL-7/DragonEye/pull/17](https://github.com/PolSl-PBL-7/DragonEye/pull/17), so I'm closing it.

**Comment by Fakser** [comment]
This PR is included in [https://github.com/PolSl-PBL-7/DragonEye/pull/17](https://github.com/PolSl-PBL-7/DragonEye/pull/17), so I'm closing it.

**Pull request closed without merging**